docs/changelog/85326.yaml

```diff
@@ -0,0 +1,24 @@
+pr: 85326
+summary: Deprecate LDAP/AD realm settings missing bind password for bind DN
+area: Authentication
+type: deprecation
+issues:
+ - 47191
+deprecation:
+  title: Deprecate LDAP/AD realm settings missing bind password for bind DN
+  area: Cluster and node setting
```

Comment:
`Authentication` and `Security` are among the possible values here, it turns out. Possibly values are here: https://github.com/elastic/elasticsearch/blob/master/build-tools-internal/src/main/resources/changelog-schema.json#L178